src/components/Main/Header.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import Cookie from 'js-cookie';
+import useHttpRequest from '../../hooks/useHttpRequest';
+import Logo from '../../images/logo.png';
+import Search from './Search';
+
+function Header () {
+  const location = useLocation();
+
+  const isCookie = Cookie.get('connect.sid');
```

Comment:
Use some meanfull names for variables... What does isCookie mean in scope of header component? Call it isUserAuthorized or something like that.

Reply:
Fixed. Thanks!